docs/source/workflow_inputs.rst

```diff
@@ -634,6 +635,7 @@ The column names available in the schedule CSV files are:
   ``hot_water_fixtures``           frac     Fixtures (sinks, showers, baths) hot water use schedule.                           Yes
   ``heating_setpoint``             F        Thermostat heating setpoint schedule.                                              No
   ``cooling_setpoint``             F        Thermostat cooling setpoint schedule.                                              No
+  ``maximum_power_ratio``          frac     Variable speed system maximum power ratio schedule.                                No
```

Comment:
I suggest adding a footnote that refers to the AHRI standard as an example of what this is for; you could borrow language from the changelog.